.tekton/basic-component-pipeline.yaml

```diff
@@ -344,24 +331,24 @@ spec:
       values: [ "false" ]
 
   - name: sast-snyk-check
+    params:
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
     runAfter:
     - clone-repository
```

Comment:
It is [uncertain to me](https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1718875883675349?thread_ts=1718296263.999339&cid=C04PZ7H0VA8) whether we should use source from `clone-repo` or from `prefetch-dependencies` but assuming the latter, `runAfter` must change.

```suggestion
  - name: sast-snyk-check
    params:
    - name: SOURCE_ARTIFACT
      value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
    runAfter:
    - prefetch-dependencies
```

Reply:
![Screenshot 2024-06-21 at 08 58 50](https://github.com/stackrox/stackrox/assets/9576848/b101f552-9d97-4869-a769-c3d7016fc592)

After removing all `runAfter` conditions, the pipeline looks like this. Given that `prefetch-dependencies` and `determine-image-tag` can now run in parallel, it LGTM.